src/js/structs/Job.js

```diff
@@ -102,7 +102,13 @@ module.exports = class Job extends Item {
   }
 
   getSchedules() {
-    return this.get('schedules') || [];
+    const schedules = this.get('schedules');
+
+    if (!Array.isArray(schedules)) {
+      return [];
```

Comment:
I agree, but doing so we are accepting an invalid API request. That's not correct.

I would vote for adding RAML (or other type of) validation to the jobs when it's available, rather than trying to be smart on what we accept.